### DISCUSSION TITLE
convert music

### DISCUSSION THREAD

**Discussion by firu82** [RandomNinjaAtk/arr-scripts#131] [General]
hello how configure for the music downloada y have fully funcional script but not find how to configure te conversion part

**Comment by ghost** [comment]
I'd like to second this please. 

I'd like to download music in FLAC, and then convert to LAME V2 before importing. The script apparently can "Can convert Downloaded FLAC files to preferred audio format and bitrate before import into Lidarr".

Is there any guidance on how to set this up? Thanks.

**Reply by RandomNinjaAtk** [reply]
Set: https://github.com/RandomNinjaAtk/arr-scripts/blob/8ae48c7941a46030bcf5be2f4962eb2dfe023302/lidarr/extended.conf#L47 to: mp3
And set https://github.com/RandomNinjaAtk/arr-scripts/blob/8ae48c7941a46030bcf5be2f4962eb2dfe023302/lidarr/extended.conf#L48 to: 320

Read the config options....

**Reply by ghost** [reply]
Right, got you, thanks... That clears things up, the comments aren't completely clear, but understood after your explanation.

Looks like the only options are CBR conversions. There's no VBR/LAME conversion e.g. V0, V1, V2 profiles possible?
